tools/go.mod

```diff
@@ -5,3 +5,7 @@ go 1.12
 require github.com/golang/protobuf v1.3.1
 
 require github.com/grpc-ecosystem/grpc-gateway v1.9.0
+
+require github.com/imdario/mergo v0.3.7
```

Comment:
We don't use go modules, you shouldn't touch this file unless you're trying to pull down a new binary tool.

Reply:
Sounds good!